### PR TITLE
[Release 6.4.1] Optimized wildcard emitter performance

### PR DESCRIPTION
**Pull request by DigitalBrainJS**
A bit of simple optimizations for a wildcard emitter, especially for cases when we emitting a simple one level event through wildcard emitter (become almost 4 times faster than before)
before:

> EventEmitter2 x 12,089,852 ops/sec ±4.00% (62 runs sampled)
> **EventEmitter2 (wild) x 167,847 ops/sec ±4.90% (63 runs sampled)**
> EventEmitter2 (wild) using plain events x 190,736 ops/sec ±3.08% (62 runs sampled)
> EventEmitter2 (wild) emitting ns x 676,378 ops/sec ±4.07% (67 runs sampled)
> EventEmitter2 (wild) emitting a plain event x 764,142 ops/sec ±3.87% (71 runs sampled)
> EventEmitter3 x 10,223,124 ops/sec ±3.51% (70 runs sampled)

after:

> Platform: win32, x64, 15267MB
> Node version: v13.11.0
> CPU: 4 x AMD Ryzen 3 2200U with Radeon Vega Mobile Gfx @ 2495MHz
> ----------------------------------------------------------------
> EventEmitterHeatUp x 3,299,321 ops/sec ±3.10% (69 runs sampled)
> EventEmitter x 3,185,748 ops/sec ±3.11% (72 runs sampled)
> EventEmitter2 x 12,538,440 ops/sec ±3.76% (65 runs sampled)
> **EventEmitter2 (wild) x 217,123 ops/sec ±9.80% (66 runs sampled)**
> **EventEmitter2 (wild) using plain events x 278,079 ops/sec ±3.80% (55 runs sampled)**
> **EventEmitter2 (wild) emitting ns x 1,717,662 ops/sec ±4.17% (70 runs sampled)
> EventEmitter2 (wild) emitting a plain event x 2,898,598 ops/sec ±3.48% (72 runs sampled)**
> EventEmitter3 x 10,730,209 ops/sec ±4.59% (63 runs sampled)
> 
> Fastest is EventEmitter2

### CODE REVIEW

**Comment by DigitalBrainJS** [comment]
> EventEmitterHeatUp x 3,397,135 ops/sec ±3.47% (71 runs sampled)
> EventEmitter x 3,728,915 ops/sec ±3.55% (74 runs sampled)
> EventEmitter2 x 12,067,339 ops/sec ±4.83% (67 runs sampled)
> **EventEmitter2 (wild) x 277,835 ops/sec ±3.57% (68 runs sampled)**
> EventEmitter2 (wild) using plain events x 382,594 ops/sec ±3.60% (67 runs sampled)
> **EventEmitter2 (wild) emitting ns x 1,725,874 ops/sec ±4.97% (66 runs sampled)
> EventEmitter2 (wild) emitting a plain event x 2,903,992 ops/sec ±4.31% (71 runs sampled)**
> EventEmitter3 x 10,763,520 ops/sec ±3.86% (65 runs sampled)
> 
> Fastest is EventEmitter2

File: test/wildcardEvents/removeAllListeners.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ module.exports= {
         ee.on('test.*', function(){
             counter++;
         });
-
+        debugger;
```
Comment (RangerMauve) [inline test/wildcardEvents/removeAllListeners.js:24]: Might want to remove this debugger statement. 😁

Comment (DigitalBrainJS) [reply test/wildcardEvents/removeAllListeners.js]: Omg. Done :) It seems we need ESLint for pre-commit here, but I'm too lazy :)

**Comment by DigitalBrainJS** [comment]
@RangerMauve It would be nice if you could complete the last step for CI integration- setup the coveralls.io service. You just need to get an environment variable for the Travis build :)

**Comment by RangerMauve** [comment]
@DigitalBrainJS Yeah sure, where do I get the veriable and where should I put it? 😁

**Comment by DigitalBrainJS** [comment]
@RangerMauve https://github.com/EventEmitter2/EventEmitter2/issues/249#issuecomment-625231761

**Comment by DigitalBrainJS** [comment]
@RangerMauve just in case 😄 :
image

**Pull request merged**